drydock/templates/kustomized/tutor15/extensions/overrides.yml

```diff
@@ -0,0 +1,132 @@
+{%- if DRYDOCK_ENABLE_RESOURCE_MANAGEMENT -%}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: lms
+spec:
+  template:
+    spec:
+      containers:
+        - name: lms
+          resources:
+            limits:
+              cpu: "{{ DRYDOCK_LMS_LIMIT_CPU }}"
+              memory: "{{ DRYDOCK_LMS_LIMIT_MEMORY }}"
+            requests:
+              cpu: "{{ DRYDOCK_LMS_REQUEST_CPU }}"
+              memory: "{{ DRYDOCK_LMS_REQUEST_MEMORY }}"
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cms
+spec:
+  template:
+    spec:
+      containers:
+        - name: cms
+          resources:
+            limits:
+              cpu: "{{ DRYDOCK_CMS_LIMIT_CPU }}"
+              memory: "{{ DRYDOCK_CMS_LIMIT_MEMORY }}"
+            requests:
+              cpu: "{{ DRYDOCK_CMS_REQUEST_CPU }}"
+              memory: "{{ DRYDOCK_CMS_REQUEST_MEMORY }}"
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: lms-worker
+spec:
+  template:
+    spec:
+      containers:
+        - name: lms-worker
+          resources:
+            limits:
+              cpu: "{{ DRYDOCK_LMS_WORKER_LIMIT_CPU }}"
+              memory: "{{ DRYDOCK_LMS_WORKER_LIMIT_MEMORY }}"
+            requests:
+              cpu: "{{ DRYDOCK_LMS_WORKER_REQUEST_CPU }}"
+              memory: "{{ DRYDOCK_LMS_WORKER_REQUEST_MEMORY }}"
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cms-worker
+spec:
+  template:
+    spec:
+      containers:
+        - name: cms-worker
+          resources:
+            limits:
+              cpu: "{{ DRYDOCK_CMS_WORKER_LIMIT_CPU }}"
+              memory: "{{ DRYDOCK_CMS_WORKER_LIMIT_MEMORY }}"
+            requests:
+              cpu: "{{ DRYDOCK_CMS_WORKER_REQUEST_CPU }}"
+              memory: "{{ DRYDOCK_CMS_WORKER_REQUEST_MEMORY }}"
+{%- if FORUM_DOCKER_IMAGE is defined %}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: forum
+spec:
+  template:
+    spec:
+      containers:
+        - name: forum
+          resources:
+            limits:
+              cpu: "{{ DRYDOCK_FORUM_LIMIT_CPU }}"
+              memory: "{{ DRYDOCK_FORUM_LIMIT_MEMORY }}"
+            requests:
+              cpu: "{{ DRYDOCK_FORUM_REQUEST_CPU }}"
+              memory: "{{ DRYDOCK_FORUM_REQUEST_MEMORY }}"
+{% endif -%}
+---
+{% endif -%}
+{%- if DRYDOCK_NEWRELIC -%}
```

Comment:
I've recently noticed a problem with this `{% endif -%}` tags, they strip the line break and the triple dash ends up at the end of the previous line (e.g. `memory: 12---`) which is incompatible yaml.

Can you replace them all with `{%- ... %}` that should avoid it. The idea behind [striping whitespace](https://jinja.palletsprojects.com/en/3.0.x/templates/#whitespace-control) was to make the diff of the files more friendly.

Reply:
Understood. Thank you  @MoisesGSalas